broadinstitute/single_cell_portal_core
# Fix label in synthetic data population and conventional metadata upload

This updates labels to match recent changes in the Experimental Factor Ontology (EFO).  As detailed in EBISPOT/efo/pull/1041, EFO changed all labels with the string "10X" to "10x", which aligns with how [10x Genomics](https://www.10xgenomics.com/) refers to itself.

This fixes a bug in synthetic data population, and a lurking bug in conventional metadata upload.

To test:
* `rails console`
* `SyntheticStudyPopulator.populate('human_lymph_node')`
* Wait a few minutes.
* Explore created study in SCP UI, verify it has visualizations (which entails conventional metadata).

This satisfies SCP-3290.
